core/common/src/main/java/alluxio/Constants.java

```diff
@@ -152,6 +152,13 @@
   public static final String UNDERFS_S3_PROXY_HOST = "alluxio.underfs.s3.proxy.host";
   public static final String UNDERFS_S3_PROXY_PORT = "alluxio.underfs.s3.proxy.port";
   public static final String UNDERFS_S3_PROXY_HTTPS_ONLY = "alluxio.underfs.s3.proxy.https.only";
+  public static final String UNDERFS_S3_ENDPOINT = "alluxio.underfs.s3.endpoint";
```

Comment:
Can you please add a comment for when a user should set those s3 endpoints?
Also, can you also update the Configuring-Alluxio-with-S3 doc to include those new configs?

Per [jets3t docs](http://www.jets3t.org/toolkit/configuration.html): The host name of the S3 service. You should only ever change this value from the default if you need to contact an alternative S3 endpoint for testing purposes.